packages/donor/src/screens/onboarding/OnboardingWizardScreenContainer.tsx

```diff
@@ -4,6 +4,7 @@ import { MainNavigationKeys } from "../../navigation/app/MainNavigationKeys";
 
 export default function OnboardingWizardScreenContainer() {
   const history = useHistory();
+  localStorage.setItem("sawWizardScreen", "true");
```

Comment:
Should you use `useEffect` to make sure this only runs once?

Also, this should run when the user *finishes* the wizard